<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JirachiAdvancer</class>
 <widget class="QWidget" name="JirachiAdvancer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>262</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Channel Jirachi Advancer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="TextBox" name="maxFrame"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Starting Seed:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="TextBox" name="currentSeed"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Max Advance:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Bruteforce Range:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="TextBox" name="targetSeed"/>
   </item>
   <item row="2" column="4">
    <widget class="TextBox" name="bruteforceRange"/>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Target Seed:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Info</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="listWidgetInfo"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="5">
    <widget class="QPushButton" name="pushButtonGenerate">
     <property name="text">
      <string>Generate</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextBox</class>
   <extends>QLineEdit</extends>
   <header>Forms/Controls/TextBox.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
